src/_scss/pages/search/filters/programSource/programSource.scss

```diff
@@ -36,8 +36,47 @@
             @import "elements/filters/_typeahead";
             .program-source-select-filter__label {
                 @include display(flex);
-                @include justify-content(space-between);
+                @import '../../../../components/visualizations/tooltip/csstooltip';
+                .tooltip-popover-container {
+                    @include display(flex);
+                    @include flex (0 0 auto);
+                    margin-left: rem(5);
+                    color: $color-gray-lighter;
+                    &:hover, &:focus {
+                        color: $color-gray-medium;
+                    }
+                    .tooltip-popover {
+                        // override the positioning to be centered over the icon
+                        bottom: rem(35);
```

Comment:
Looks like this value isn't always reliable with the first and third icons. These are the icons which have to the left of them a <label> component that overflows/wraps to the second line like so:

Allocation Transfer Agency Identifier
(ATA)
Beginning Period of Availability
(BPOA)

I would guess this is the root cause as all the other icons work perfectly and the <label> to the left does not wrap.

Reply:
Good catch. I changed the icon alignment so that it's not totally obscured, since dynamically measuring seems overkill.